Jenkinsfile

```diff
@@ -9,6 +9,12 @@ node('linux && docker') {
         sh 'npm run setup'
       }
 
+      if (isMaster) {
+        stage('Bump build version') {
+          sh 'npx lerna version --conventional-prerelease --amend'
+        }
```

Comment:
The version isn't dynamically read by headless nor atomic, it's added at build time. Therefor, headless and atomic must be built before the version is bumped.

Reply:
I think this would be a useful comment to include in the file. I can see myself in a few months wondering what this stage is doing.